crates/metrics/src/buckets.rs

```diff
@@ -0,0 +1,164 @@
+use std::{
+    collections::HashMap,
+    sync::OnceLock,
+};
+#[cfg(test)]
+use strum_macros::EnumIter;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(test, derive(EnumIter))]
+pub(crate) enum Buckets {
+    Timing,
+    GasUsed,
+    TransactionsCount,
+    Fee,
+    SizeUsed,
+}
+static BUCKETS: OnceLock<HashMap<Buckets, Vec<f64>>> = OnceLock::new();
+pub(crate) fn buckets(b: Buckets) -> impl Iterator<Item = f64> {
+    BUCKETS.get_or_init(initialize_buckets)[&b].iter().copied()
+}
+
+#[rustfmt::skip]
+fn initialize_buckets() -> HashMap<Buckets, Vec<f64>> {
```

Comment:
I think we need to keep a histogram only for timing. 

A histogram for timing is useful because requests or their time of execution is random, and we can't track data about them using just numbers. The histogram here provides a view of many events with different properties that happened during some period of time. But in the case of statistics about the block, it's always the same. Each block has a known value and we know taht the block is generated each second, time here is our axis that allows us to create a histogram later

Reply:
Yes, I think it makes sense

Since each block will have a fixed number of transactions, it'll be _const value_ for the entire monitored slot (1 sec.) So if we report this single number to registry, we can later get every stat we need from Grafana

This applies to: `importer_gas_per_block`, `importer_fee_per_block_gwei` and `importer_transactions_per_block`. I'm gonna update the PR.

Reply:
Updated in: https://github.com/FuelLabs/fuel-core/pull/2310/commits/c043ed7a7a3aa1226bcd933a7f0b9c2a12b5538d

Reply:
Also, gas price metric added in https://github.com/FuelLabs/fuel-core/pull/2310/commits/7d17f992e104eb3e9e15c9efcdd6071f387ba363